src/components/Team/CardContainer.jsx

```diff
@@ -0,0 +1,272 @@
+                        <MemberCard name="Paul Carney" title="Hardware Lead Engineer"
+                        gradYear="2022"
+                        interests="Skateboarding, guitar, drums"
+                        funFact="Can kickflip over you"
+                        githubUrl=""
+                        instagramUrl=""
+                        linkedinUrl="https://www.linkedin.com/in/paul-carney-07a273169/"
+                        facebookUrl="https://www.facebook.com/paul.carney.319?ref=bookmarks"
+                        personalUrl=""
+                        imagePath='/memberPhotos/Paul.jpeg'  />
+                    </div>
+                    <div className="cardContainerColumn">
+                        <MemberCard name="Thomas Smith" title="iOS Lead Engineer"
+                        gradYear="2022"
+                        interests="Backpacking, canoeing, board games"
+                        funFact="Walks 10+ miles every day"
+                        githubUrl="https://github.com/thomasebsmith/"
+                        instagramUrl=""
+                        linkedinUrl="https://www.linkedin.com/in/thomasebsmith/"
+                        facebookUrl=""
+                        personalUrl="https://thomasebsmith.github.io/"
+                        imagePath='/memberPhotos/Thomas.png'  />
+                    </div>
+                    <div className="cardContainerColumn">
+                        <MemberCard name="Eric Lian" title="React Native Lead Engineer"
+                        gradYear="2022"
+                        interests="Astronomy, Drawing, Music (guitar, piano), Filmmaking, Workout, Stocks, Video Games, cards & board games"
+                        funFact="Skipped kindergarden"
+                        githubUrl="https://github.com/ericlian1"
+                        instagramUrl="https://www.instagram.com/eric_lian_/"
+                        linkedinUrl="https://www.linkedin.com/in/ericlian419/"
+                        facebookUrl="https://www.facebook.com/EricLian123"
+                        personalUrl=""
+                        imagePath='/memberPhotos/Eric.JPG'  />
+                    </div>
+                    <div className="cardContainerColumn">
+                        <MemberCard name="Junlin Mo" title="Interviewing Director"
+                        gradYear="2021"
+                        interests="Business, Music (piano, guitar, writing, singing), Cognition, Freerunning, Fitness, Video games"
+                        funFact="Started a few businesses, also an EECS 281 IA"
+                        githubUrl=""
+                        instagramUrl=""
+                        linkedinUrl="https://www.linkedin.com/in/junlin-mo/"
+                        facebookUrl=""
+                        personalUrl=""
+                        imagePath="/memberPhotos/Junlin.jpg"  />
+                    </div>
+                    <div className="cardContainerColumn">
```

Comment:
A bit unfortunate that we don't have these, are we sure we want to ship this with this placeholder info?